lib/ansible/modules/windows/win_domain.ps1

```diff
@@ -29,17 +29,28 @@ $domain_netbios_name = Get-AnsibleParam -obj $params -name "domain_netbios_name"
 $safe_mode_admin_password = Get-AnsibleParam -obj $params -name "safe_mode_password" -failifempty $true
 $database_path = Get-AnsibleParam -obj $params -name "database_path" -type "path"
 $sysvol_path = Get-AnsibleParam -obj $params -name "sysvol_path" -type "path"
+$create_dns_delegation = Get-AnsibleParam -obj $params -name "create_dns_delegation" -type "bool" -default $true
+$domain_mode = Get-AnsibleParam -obj $params -name "domain_mode" -type "str" -choices "Win2003","Win2008","Win2008R2","Win2012","Win2012R2","WinTreshold"
```

Comment:
Typo: WinTreshold should be WinThreshold (also on L34) - at least if the docs here are correct: https://docs.microsoft.com/en-us/powershell/module/addsdeployment/install-addsforest?view=win10-ps

Reply:
Good catch !

Reply:
Fixed.